### ISSUE TITLE
still looking for that hardsub OCRING feature :)

### ISSUE THREAD

**Issue by techguru0** [SubtitleEdit/subtitleedit#8874]
hopefully one day
until it does get added
for anyone who might need to do this
there is this program
https://github.com/YaoFANGUK/video-subtitle-extractor
it works decently with english and chinese

**Comment by Masina86** [comment]
Yes it would be cool if subtitle edit has this but not only for 2 languages, check videosubfinder it may help ya after 10 years i think it it still best hardsub to images program u can find nowdays.
